resources/keywords/dbaas_policy.resource

```diff
@@ -39,25 +42,32 @@ Create Custom Namespace
     ${lblProjectSelectUpd}=     Replace String    ${lblProjectSelect}    <<project>>    ${custom_ns}
     Wait Until Page Contains Element    ${lblProjectSelectUpd}
 
-Setup DBaaSPolicy for ${namespace}
+Setup DBaaSPolicy for ${namespace} as ${DBaaSPersona}
     User Navigates To DBaaSPolicy Screen
     Run Keyword If    """.11""" in """${CLUSTER_VERSION}"""
     ...    Click Element    ${lblOperand}
     Comment    The above step Click Element ${lblOperand} should be enabled
     ...        only for Openshift Version >= 4.11
     ...        IF statement can be removed once support for 4.10 is dropped
     ${status} =    Run Keyword And Return Status    Wait Until Page Contains Element    ${txtNoDbaaSPolicy}    timeout=7s
-    Run Keyword If    ${status}
-    ...    Create Custom DBaaSPolicy
+    IF    ${status}
+        Create Custom DBaaSPolicy as ${DBaaSPersona}
+        DBaaSPolicy Creation Success
+    END
     Set Global Variable    ${DBaaSPolicyChecked}    ${True}
 
-Create Custom DBaaSPolicy
-    [Arguments]    ${persona}=${DBAASPOLICY.PERSONA}
-    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
-    Click Element    ${btnCreateDBaaSPolicy}
+DBaaSPolicy Creation Success
+    Click If Element Present    ${lblOperand}
+    ${lnkDBaaSPolicyUpd} =    Replace String    ${lnkDBaaSPolicyName}    <<policyname>>   ${policyName}
+    Wait Until Page Contains Element    ${lnkDBaaSPolicyUpd}    timeout=10s
+
+Create Custom DBaaSPolicy as ${DBaaSPersona}
     Wait Until Element Is Visible    ${titleCreateDBaaSPolicy}    timeout=10s
+    Click Element    ${radFormView}
     Element Should Be Visible    ${txtBxDBaaSPolicyName}
-    ${policyName} =    Set Variable   ${persona}-policy
+    ${persona}=     Remove String    ${DBaaSPersona}     _
```

Comment:
Why are we removing the DBaaS Persona string here?

Reply:
The "Remove String" keyword would remove the substring specified for on the given string. In the above example we are removing the _ [underscore] from the variable ${DBaaSPersona}. So, if the ${DBaaSPersona} is "OCP_ADMIN" then value on ${persona} would be "OCPADMIN"